test/integration/targets/apt/tasks/url-with-deps.yml

```
@@ -48,9 +48,37 @@
         - dpkg_result is successful
         - dpkg_result.rc == 0
 
+    - name: Install deb file with recommends from URL
+      apt:
+        deb: https://github.com/Akasurde/git_demo/raw/master/bar.deb
```

this should work now
```suggestion
        deb: https://ci-files.testing.ansible.com/test/integration/targets/apt/bar.deb
```